### PR TITLE
Fix IIS benchmarks

### PR DESCRIPTION
**Pull request by shirhatti**

### CODE REVIEW

File: src/Microsoft.Crank.Agent/Startup.cs [hunk 1]
```diff
@@ -91,7 +92,7 @@ public class Startup
         private static readonly HashSet<string> _installedSdks = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
 
         private const string _defaultUrl = "http://*:5010";
-        private static readonly string _defaultHostname = Environment.MachineName.ToLowerInvariant();
+        private static readonly string _defaultHostname = Dns.GetHostName();
```
Comment (shirhatti) [inline src/Microsoft.Crank.Agent/Startup.cs:95]: If your machine name is longer than 15 characters, it won't match the hostname.

**Comment by shirhatti** [comment]
FYI @jkotalik

**Comment by sebastienros** [comment]
Do you want me to deploy this asap to test the scenarios?

**Comment by shirhatti** [comment]
@sebastienros Nothing urgent. Can you ping me whenever you do end up deploying it?

**Comment by jkotalik** [comment]
Yay!

**Comment by davidfowl** [comment]
What does this fix?

**Comment by shirhatti** [comment]
```
crank --config .\scenarios\json.benchmarks.yml --scenario json --profile aspnet-perf-win --variable server=IISInProcess --application.webhost IISInProcess

[11:33:21.189] Running session '804ecc78149d48e48c2694075f66aab2' with description ''
[11:33:21.268] Starting job 'application' ...
[11:33:21.282] Submitted job: http://asp-perf-win:5001/jobs/2
[11:33:41.629] Job has been selected by the server ...
[11:33:41.632] Using local folder: "C:\Users\shirh\source\repos\Benchmarks\"
[11:33:44.056] Uploading C:\Users\shirh\AppData\Local\Temp\tmpB91D.tmp
[11:33:45.050] Job is now building ... http://asp-perf-win:5001/jobs/2/buildlog
[11:34:38.858] Job is running ... http://asp-perf-win:5001/jobs/2/output
[11:34:41.170] Starting job 'load' ...
[11:34:41.181] Submitted job: http://asp-perf-db:5001/jobs/23
[11:34:42.216] Job has been selected by the server ...
[11:34:42.225] Job is now building ... http://asp-perf-db:5001/jobs/23/buildlog
[11:34:50.363] Job is running ... http://asp-perf-db:5001/jobs/23/output
[11:35:20.828] Stopping job 'load' ...
[11:35:21.899] Deleting job 'load' ...
[11:35:21.916] Stopping job 'application' ...
[11:35:29.053] Deleting job 'application' ...

| application           |                                                                   |
| --------------------- | ----------------------------------------------------------------- |
| CPU Usage (%)         | 85                                                                |
| Cores usage (%)       | 1,025                                                             |
| Working Set (MB)      | 169                                                               |
| Build Time (ms)       | 6,545                                                             |
| Start Time (ms)       | 30,017                                                            |
| Published Size (KB)   | 105,608                                                           |
| .NET Core SDK Version | 5.0.101                                                           |
| ASP.NET Core Version  | 5.0.1+fc93e595ceffbb1e3e85532bf454e92a6a80dd6b                    |
| .NET Runtime Version  | 5.0.1-servicing.20575.16+b02e13abab3d6f499751af2ad1ae6333fb4e633d |


| load                   |           |
| ---------------------- | --------- |
| CPU Usage (%)          | 48        |
| Cores usage (%)        | 581       |
| Working Set (MB)       | 41        |
| Build Time (ms)        | 4,942     |
| Start Time (ms)        | 126       |
| Published Size (KB)    | 77,077    |
| .NET Core SDK Version  | 3.1.404   |
| First Request (ms)     | 319       |
| Requests               | 2,318,449 |
| Bad responses          | 0         |
| Mean latency (us)      | 1,651     |
| Max latency (us)       | 45,871    |
| Requests/sec           | 154,954   |
| Requests/sec (max)     | 166,447   |
| Read throughput (MB/s) | 24.03     |
```

**Comment by davidfowl** [comment]
Collect a profile to make sure it's really using IIS, though I bet you'll find it'll probably fail because of permissions or something...

**Pull request merged**